wiki/graphql-api.md

```diff
@@ -0,0 +1,200 @@
+## GraphQL API
+
+We expose a GraphQL API for universal drug codes using [Apollo Server](https://github.com/apollographql/apollo-server#readme). 
```

Comment:
Since this is documenting a service, I think maybe better to just focus on the API (hide implementation details like Apollo etc.)? I think the user will mostly just want to know how to get the data they want, implementation detail is probably more suitable when documenting the architecture or contributor guides etc.

Reply:
So.. the wiki is more to explain how to format queries and how the data is structured?
We should create an area for more under-the-hood explanations somewhere else?

Reply:
I would still suggest having the implementation details in the wiki, just not in the top-level section, as users will just want to know how it works, rather than whats under-the-hood. I think implementation details might be nice to be left to a section for contributors/"building from source"?

Reply:
Cool. So this means top-level things will be end-user focused (user guides/query and response structure) and I'll move the details (including the vscode config file) into a separate section for developers/contributors?

Reply:
![thumbsup](https://user-images.githubusercontent.com/43223637/88866123-ceb54880-d25d-11ea-9809-d72a8d2b361c.gif)